package.json

```diff
@@ -37,26 +37,40 @@
     "url": "https://github.com/iTwin/insights-client"
   },
   "dependencies": {
-    "cross-fetch": "^3.1.4"
+    "@azure/storage-blob": "^12.11.0",
+    "@bentley/oidc-signin-tool": "^2.19.28",
```

Comment:
Upgrade to 3.x (https://www.npmjs.com/package/@itwin/oidc-signin-tool)

Reply:
upgraded to 3.4.1